dist/api_data.py

```diff
@@ -659,6 +659,9 @@ def __ge__(self, other):
         ]),
     Config('error_prefix', '', r'''
         prefix string for error messages'''),
+    Config('event_handler_json', 'false', r'''
+        if true, encode event handler messages in JSON format.''',
+        type='boolean'),
```

Comment:
One consideration is the naming. Do we ever think that other handlers like msg_handler, panic_handler or other calls might want JSON too? It seems possible to me even if it is not needed today. What if we used something like the old-style verbose mechanism where, at first, only `event` would be supported. Eventually it might look something like:
json=(event,panic)

Now would be the time to set up something like this if we're ever going to do so.

Reply:
We've iterated on your feedback and re-worked the option to use a list, where each option corresponds to an event handler category e.g. 'json_output=[message,error]'. Let us know if this looks OK :)